src/content/docs/apis/intro-apis/introduction-new-relic-apis.mdx

```diff
@@ -72,20 +72,77 @@ This document provides examples and reference information for our API endpoints.
   To use APIs and SDKs, as well as the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
+
+## Ingest APIs for core data types [#data-type-apis]
+
+We have [many solutions for reporting data](/docs/telemetry-data-platform/get-started/introduction-new-relic-data-ingest-apis-sdks), and our APIs are one of these ways. These APIs can be used directly, but they're also the underlying ingest route for many of our more curated tools, such as our [open source telemetry integrations](/docs/integrations/open-source-telemetry-integrations/get-started/introduction-new-relics-open-source-telemetry-integrations) and our [Telemetry SDKs](/docs/telemetry-data-platform/ingest-apis/telemetry-sdks-report-custom-telemetry-data). 
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Data type
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        [Trace API](/docs/apm/distributed-tracing/trace-api/introduction-new-relic-trace-api)
```

Comment:
Why not order this table so that it goes:
Metrics 
Events 
Logs
Traces? MELT is such a fun acronym.

Are we really wanting to emphasize that Traces API by putting it first?

Reply:
one thing that helps explain this approach is that the Trace API and Event API are basically the only two ingest APIs likely to be used on their own. Metric API and Log API will hardly ever be used directly by customers. i prob need to make that a little more clear....